6_visualize/src/create_gif_tasks.R

```diff
@@ -32,23 +56,27 @@ create_gif_tasks <- function(timestep_ind, folders, view_ind, basemap_ind, storm
     },
     command = function(task_name, ...){
       cur_task <- dplyr::filter(rename(tasks, tn=task_name), tn==task_name)
+
       psprintf(
         "create_storm_frame(",
-        "png_file=target_name",
+        "png_file=target_name,",
```

Comment:
I guess we should rename this step from `gif_frame` to `png_frame` or `animation_frame` to be accurate.